osu.Game/Overlays/Settings/Sections/Input/TabletSettings.cs

```diff
@@ -110,11 +110,10 @@ private void load(OsuColour colours)
                             if (RuntimeInfo.OS == RuntimeInfo.Platform.Windows || RuntimeInfo.OS == RuntimeInfo.Platform.Linux)
                             {
                                 t.NewLine();
-                                t.AddText("If your tablet is not detected, please read ");
-                                t.AddLink("this FAQ", LinkAction.External, RuntimeInfo.OS == RuntimeInfo.Platform.Windows
+                                var formattedSource = MessageFormatter.FormatText(localisation.GetLocalisedBindableString(TabletSettingsStrings.NoTabletDetectedDescription(RuntimeInfo.OS == RuntimeInfo.Platform.Windows
```

Comment:
Very unsure about this direction. Is there precedent elsewhere (osu-web or other localised strings)?

Reply:
I tried to localize it in MessageFormatter because it contains a link in the middle of a string.
I have not seen any particular precedent...

Reply:
Looking at the osu-web code, it seems to localize using regular expressions... I used Chat's MessageFormatter to include the link this time, but should I follow osu-web and use regular expressions?
Since I only want to include the link in this case, it looks smarter to do it in MessageFormatter...
https://github.com/ppy/osu-web/blob/28bb42f994996bcf31c97e6d6ebb195a6a94c034/app/Models/Event.php#L37

Reply:
Hmm, based on how we have things setup this does look to be the best way forward. Let's go with it for now.